src/store/thunk.js

```diff
@@ -0,0 +1,48 @@
+import {
```

Comment:
Please add 'thunk' to the names of the functions

Reply:
Fixed